core/src/main/data/scripts/repofilter/default-pattern-repofilter.groovy

```diff
@@ -0,0 +1,25 @@
+package org.commonjava.indy.core.content.group
+
+import org.commonjava.indy.model.core.Group
+
+class DefaultPatternNameGroupRepositoryFilter extends PatternNameGroupRepositoryFilter {
```

Comment:
Seems I did not find this PatternNameGroupRepositoryFilter. Should it be ReversePatternNameGroupRepositoryFilter?

Reply:
Yeah, this is mistake from earlier code. I will remove this file.